src/software/jetson_nano/services/robot_auto_test.cpp

```diff
@@ -0,0 +1,109 @@
+#include <chrono>
+#include <thread>
+
+#include "proto/message_translation/tbots_geometry.h"
+#include "proto/primitive/primitive_msg_factory.h"
+#include "shared/2021_robot_constants.h"
+#include "shared/constants.h"
+#include "software/jetson_nano/primitive_executor.h"
+#include "software/jetson_nano/services/motor.h"
+#include "software/jetson_nano/services/power.h"
+#include "software/util/scoped_timespec_timer/scoped_timespec_timer.h"
+
+extern "C"
+{
+#include "external/trinamic/tmc/ic/TMC4671/TMC4671.h"
+#include "external/trinamic/tmc/ic/TMC4671/TMC4671_Variants.h"
+#include "external/trinamic/tmc/ic/TMC6100/TMC6100.h"
+}
+
+std::unique_ptr<MotorService> motor_service_;
+std::unique_ptr<PowerService> power_service_;
+RobotConstants_t robot_constants_;
+int read_value;
+
+// SPI Chip Selects
+static const uint8_t FRONT_LEFT_MOTOR_CHIP_SELECT =
+    motor_service_->FRONT_LEFT_MOTOR_CHIP_SELECT;
+static const uint8_t FRONT_RIGHT_MOTOR_CHIP_SELECT =
+    motor_service_->FRONT_RIGHT_MOTOR_CHIP_SELECT;
+static const uint8_t BACK_LEFT_MOTOR_CHIP_SELECT =
+    motor_service_->BACK_LEFT_MOTOR_CHIP_SELECT;
+static const uint8_t BACK_RIGHT_MOTOR_CHIP_SELECT =
+    motor_service_->BACK_RIGHT_MOTOR_CHIP_SELECT;
```

Comment:
do you need to redefine these constants? you can't use it as MotorService::BACK_LEFT_MOTOR_CHIP_SELECT

Reply:
Right it's redundant, will use the constants as defined in `MotorService`